app/controllers/sites/spam_protections_controller.rb

```diff
@@ -8,10 +8,9 @@ def edit
 
   def update
     if @settings.update(params[:settings])
-      flash[:notice] = 'Bot protection settings updated.'
-      redirect_to edit_admin_site_spam_protection_url
+      redirect_to edit_admin_site_spam_protection_url, t('.success')
```

Comment:
```suggestion
      redirect_to edit_admin_site_spam_protection_url, success: t('.success')
```

Otherwise it's broken :upside_down_face: